css/style.css

```diff
@@ -54,6 +62,150 @@ body {
   margin: auto;
 }
 
+main,
+header,
+article,
+aside,
+footer,
+section {
+  padding: 5px;
+  float: left;
+}
+
+header {
+  width:100%;
+  background-color: #033048;
+  padding: 40px;
+}
+
+
+
+nav {
+  float: right;
+  font-size: 24px;
+  font-weight: 600;
+  color: yellow;
+  padding: 10px;
+}
+
+
+main {
+  width: 710px;
+  overflow: auto;
+}
+
+article {
+  color: #033048;
+}
+
+h2{
+  font-size: 40px;
+  font-weight: 600;
+  color: #033048;
+  padding: 10px;
+}
+
+h3 {
+  font-size: 32px;
+  font-weight: 800;
+  padding: 5px;
+}
+
+em {
+  font-weight: bold;
+}
+
+ul {
+  padding: 5px;
+}
+
+h5 {
+  font-size: 24px;
+  font-weight: 600;
+  color: #033048;
+}
+
+p {
+  font-size: 24px;
+  font-weight: 400;
+  padding: 5px;
+
+}
+
+.article-end {
+  font-weight: 800;
+  float: right;
+}
+
+aside {
+  width: 240px;
+  min-height: 75px;
+  background-color: lightgrey;
+  color: #033048;
+  padding: 30px;
+  overflow: auto;
+}
+
+h3 {
+  font-size: 32px;
+  font-weight: 800;
+}
+
+p {
+  font-size: 24px;
+  font-weight: 400;
+}
+
+section {
+  background-color: #f9e42e;
+  width: 100%;
```

Comment:
once you remove the global `float` rule you have at the top, you can remove the `width: 100%` and instead do a `clear: both;`. That will clear the floats above it on the `main` and `aside`